pkg/daemon/daemon.go

```diff
@@ -724,6 +701,58 @@ func (dn *Daemon) nodeStateSyncHandler() error {
 	return nil
 }
 
+func (dn *Daemon) skipReconciliation(latestState *sriovnetworkv1.SriovNetworkNodeState) (bool, error) {
+	var err error
+
+	// Skip when SriovNetworkNodeState object has just been created.
+	if latestState.GetGeneration() == 1 && len(latestState.Spec.Interfaces) == 0 {
+		err = dn.storeManager.ClearPCIAddressFolder()
+		if err != nil {
+			log.Log.Error(err, "failed to clear the PCI address configuration")
+			return false, err
+		}
+
+		log.Log.V(0).Info(
+			"nodeStateSyncHandler(): interface policy spec not yet set by controller for sriovNetworkNodeState",
+			"name", latestState.Name)
+		if latestState.Status.SyncStatus != "Succeeded" {
+			dn.refreshCh <- Message{
+				syncStatus:    "Succeeded",
+				lastSyncError: "",
+			}
+			// wait for writer to refresh status
+			<-dn.syncCh
+		}
+		return true, nil
+	}
+
+	// Verify changes in the spec of the SriovNetworkNodeState CR.
+	if dn.nodeState.GetGeneration() == latestState.GetGeneration() {
+		genericPlugin, ok := dn.enabledPlugins[GenericPluginName]
+		if ok {
+			// Verify changes in the status of the SriovNetworkNodeState CR.
+			if genericPlugin.CheckStatusChanges(latestState) {
+				return false, nil
+			}
+		}
+
+		log.Log.V(0).Info("nodeStateSyncHandler(): Interface not changed")
```

Comment:
log msg is wrong here

Reply:
That was already there. But I don't think it is completely wrong, perhaps incomplete. It logs that the Interface parameter of the spec didn't change.

Reply:
calling method is different...